book/validation.rst

```diff
@@ -777,33 +801,31 @@ user registers and when a user updates their contact information later:
 
     .. code-block:: php
 
-        // src/Acme/BlogBundle/Entity/User.php
-        namespace Acme\BlogBundle\Entity;
+        // src/AppBundle/Entity/User.php
+        namespace AppBundle\Entity;
 
         use Symfony\Component\Validator\Mapping\ClassMetadata;
-        use Symfony\Component\Validator\Constraints\Email;
-        use Symfony\Component\Validator\Constraints\NotBlank;
-        use Symfony\Component\Validator\Constraints\Length;
+        use Symfony\Component\Validator\Constraints as Assert;
 
         class User
         {
             public static function loadValidatorMetadata(ClassMetadata $metadata)
             {
-                $metadata->addPropertyConstraint('email', new Email(array(
+                $metadata->addPropertyConstraint('email', new Assert\Email(array(
                     'groups' => array('registration'),
                 )));
 
-                $metadata->addPropertyConstraint('password', new NotBlank(array(
+                $metadata->addPropertyConstraint('password', new Assert\NotBlank(array(
                     'groups' => array('registration'),
                 )));
-                $metadata->addPropertyConstraint('password', new Length(array(
-                    'min'  => 7,
-                    'groups' => array('registration')
+                $metadata->addPropertyConstraint('password', new Assert\Length(array(
+                    'min'    => 7,
+                    'groups' => array('registration'),
                 )));
 
-                $metadata->addPropertyConstraint(
-                    'city',
-                    Length(array("min" => 3)));
+                $metadata->addPropertyConstraint('city', Assert\Length(array(
```

Comment:
This is not correct, see https://github.com/symfony/symfony-docs/pull/4789 (including some other fixes)